routers/credential.py

```diff
@@ -0,0 +1,42 @@
+import json
+from fastapi import APIRouter, Response, status
+
+from schemas import SecuredCredentialWithOptions
+from schemas.mappings.vcr_credential import VCRCredential
+from services import vcr as vcr_service
+
+router = APIRouter(prefix="/credentials", tags=["credentials"])
+
+
+@router.post(
+    "/",
+    status_code=status.HTTP_201_CREATED,
+    response_model=SecuredCredentialWithOptions,
+    response_model_exclude_none=True,
+    response_model_by_alias=True,
+)
+async def issue_credential(secured_credential: SecuredCredentialWithOptions):
+    """Issue a new credential"""
```

Comment:
```suggestion
async def receive_credential(secured_credential: SecuredCredentialWithOptions):
    """Receive a new credential"""
```

The term issuing is misleading here as you are receiving already issued credentials